sdk/communication/Azure.Communication.PhoneNumbers/CHANGELOG.md

```diff
@@ -2,13 +2,8 @@
 
 ## 1.1.0-beta.2 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
 ### Other Changes
+- 'Deprecates' (read discourage) the use of a method from the public API
```

Comment:
I'd specify which method is marked here.

Reply:
This is a good call